lib/src/sidenav/Sidenav.tsx

```diff
@@ -67,49 +156,156 @@ const SidenavContainer = styled.div<SidenavPropsType>`
   }
 `;
 
-const SidenavMenuTitle = styled.div`
-  font-family: ${(props) => props.theme.titleFontFamily};
-  font-size: ${(props) => props.theme.titleFontSize};
-  font-style: ${(props) => props.theme.titleFontStyle};
-  font-weight: ${(props) => props.theme.titleFontWeight};
-  color: ${(props) => props.theme.titleFontColor};
-  letter-spacing: ${(props) => props.theme.titleFontLetterSpacing};
-  text-transform: ${(props) => props.theme.titleFontTextTransform};
-  margin-bottom: 16px;
+const SidenavTitle = styled.div`
+  font-family: "Open Sans";
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 27px;
+  display: flex;
+  align-items: center;
+
+  color: #4d4d4d;
+  padding: 12px 24px;
+
+  svg {
+    margin-right: 8px;
+  }
 `;
 
-const SidenavMenuSubTitle = styled.div`
-  font-family: ${(props) => props.theme.subtitleFontFamily};
-  font-size: ${(props) => props.theme.subtitleFontSize};
-  font-style: ${(props) => props.theme.subtitleFontStyle};
-  font-weight: ${(props) => props.theme.subtitleFontWeight};
-  color: ${(props) => props.theme.subtitleFontColor};
-  letter-spacing: ${(props) => props.theme.subtitleFontLetterSpacing};
-  text-transform: ${(props) => props.theme.subtitleFontTextTransform};
-  margin-bottom: 4px;
+const SidenavGroup = styled.div`
+  width: 100%;
+  a {
+    padding: 7px 24px 7px 36px;
+  }
 `;
 
-const SidenavMenuLink = styled.a`
-  font-family: ${(props) => props.theme.linkFontFamily};
-  font-size: ${(props) => props.theme.linkFontSize};
-  font-style: ${(props) => props.theme.linkFontStyle};
-  font-weight: ${(props) => props.theme.linkFontWeight};
-  color: ${(props) => props.theme.linkFontColor};
+const SidenavGroupTitle = styled.span`
+  svg {
+    width: 16px;
+    margin-right: 8px;
+  }
+  box-sizing: border-box;
+  width: 100%;
+
+  font-family: ${(props) => props.theme.groupTitleFontFamily};
+  font-style: ${(props) => props.theme.groupTitleFontStyle};
+  font-weight: ${(props) => props.theme.groupTitleFontWeight};
+  font-size: ${(props) => props.theme.groupTitleFontSize};
+  line-height: 19px;
+
+  display: flex;
+  align-items: center;
+  margin: 0px;
+  padding: 7px 24px;
+`;
```

Comment:
Same here. Why this value for the padding? We never use odd values, like 7px.

Reply:
Yes, you are right I checked the wrong measurements. I added 1 px from the outline in the design.

I will also change the values from px to rem but maintaining the same value and change the 7px.